concent_api/utils/logging.py

```diff
@@ -100,6 +100,24 @@ def log_400_error(endpoint: str, message: Message, client_public_key: str):
     ))
 
 
+def log_no_changes_in_subtask_states(client_public_key: str):
```

Comment:
starting from python 3.6, you can use f-strings ;)
https://www.python.org/dev/peps/pep-0498/

Reply:
On cluster we still don't have python 3.6, we have to wait until it will be available for us :)